user_guide/src/examples/expressions/window.py

```diff
@@ -1,7 +1,7 @@
 from .dataset import df
 import polars as pl
 
-df = df[
+out = df[
```

Comment:
We import these in expressions/__main__.py so we need to change the name there as well.